src/main/java/org/protege/editor/owl/server/api/Server.java

```diff
@@ -1,6 +0,0 @@
-package org.protege.editor.owl.server.api;
-
-public interface Server extends ServerServices {
-
-    // NO-OP
-}
```

Comment:
this was my favorite interface and I always wondered if these empty interfaces you see in Java are merely a way to put a name on something?